.travis.yml

```diff
@@ -22,6 +23,9 @@ matrix:
   - rvm: 2.4.4
     bundler_args: --without system_tests development release
     env: PUPPET_VERSION="~> 5.0" CHECK=build DEPLOY_TO_FORGE=yes
+  - rvm: 2.5.1
+    bundler_args: --without development release
+    env: CHECK=acceptance
```

Comment:
all the changes in this file need to go in the sync yaml

Reply:
I manually just synced the added cell without using `pdk update`...is that ok?

Reply:
We don't yet use pdk, we currently use https://github.com/voxpupuli/modulesync_config/ as template for our modules.